cmd/multiclusterapp.go

```diff
@@ -803,6 +1030,46 @@ func outputMultiClusterAppRevisions(ctx *cli.Context, c *cliclient.MasterClient)
 	return writer.Err()
 }
 
+func outputMultiClusterAppMembers(ctx *cli.Context, c *cliclient.MasterClient, app *managementClient.MultiClusterApp) error {
```

Comment:
I think this should be its own command `list-members` following what is done in project. The `show-app` is about version info and if there are a lot of users on that app it's going to make for a large output.

Reply:
Do you expect a `list-roles` command to display roles info of a mcapp

Reply:
nevermind, just saw your comment below